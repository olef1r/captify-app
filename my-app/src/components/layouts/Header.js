
import React from 'react';
import { makeStyles } from '@material-ui/core/styles';
import Toolbar from '@material-ui/core/Toolbar';
import Typography from '@material-ui/core/Typography';
import Container from '@material-ui/core/Container';

const useStyles = makeStyles(theme => ({
  toolbar: {
    borderBottom: `1px solid ${theme.palette.divider}`,
  },
  toolbarTitle: {
    flex: 1,
  },
}));

export default function Blog() {
  const classes = useStyles();
  return (
    <Container maxWidth="lg">
        <Toolbar className={classes.toolbar}>
          <Typography
            component="h2"
            variant="h5"
            color="inherit"
            align="center"
            noWrap
            className={classes.toolbarTitle}
          >
            There is data from DB
          </Typography>
        </Toolbar>
  </Container>
)};
  