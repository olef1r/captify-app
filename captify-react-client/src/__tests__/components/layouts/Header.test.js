import React from 'react';
import { shallow } from 'enzyme';
import Header from '../../../components/layouts/Header';

let wrapper;

beforeEach(() => {
  wrapper = shallow(<Header/>);
});

describe('<Header> - Component', () => {
  it('renders without crashing', () => {
    expect(wrapper).toMatchSnapshot();
    expect(wrapper).toIncludeText('There is data from DB')
  });
});
