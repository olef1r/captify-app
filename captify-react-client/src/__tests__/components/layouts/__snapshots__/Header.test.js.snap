// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header> - Component renders without crashing 1`] = `
<WithStyles(ForwardRef(Container))
  maxWidth="lg"
>
  <WithStyles(ForwardRef(Toolbar))
    className="makeStyles-toolbar-1"
  >
    <WithStyles(ForwardRef(Typography))
      align="center"
      className="makeStyles-toolbarTitle-2"
      color="inherit"
      component="h2"
      noWrap={true}
      variant="h5"
    >
      There is data from DB
    </WithStyles(ForwardRef(Typography))>
  </WithStyles(ForwardRef(Toolbar))>
</WithStyles(ForwardRef(Container))>
`;
